pay/contracts/interfaces/IHectorRefund.sol

```diff
@@ -0,0 +1,13 @@
+// SPDX-License-Identifier: AGPL-3.0-or-later
+pragma solidity ^0.8.17;
+
+interface IHectorRefund {
+    struct Refund {
+        uint48 limitPeriod;
+        uint48 percent;
```

Comment:
The Refund data will be prepared for each subscription plan so we can calculate the percent value from usd_amount refund and plan_usd_price

Why I have implemented percent value:
Because it would be simple & easy-to-understand on the smart contract to calculate the `refund_token_amount`
If we use the `refund_usd_amount`, then need to convert it to the token_amount again